src/CustomerBundle/Twig/Extension/AddressExtension.php

```diff
@@ -81,7 +80,7 @@ public function renderAddress(\Twig_Environment $environment, $address, $showNam
         $requiredAddressKeys = ['firstname', 'lastname', 'address1', 'postcode', 'city', 'country_code'];
 
         if (!($address instanceof AddressInterface) && (!\is_array($address) || 0 !== \count(array_diff($requiredAddressKeys, array_keys($address))))) {
-            throw new InvalidParameterException(sprintf(
+            throw new \RuntimeException(sprintf(
```

Comment:
@core23 , can you confirm `InvalidParameterException` is deprecated in favor of `\RuntimeException`?

Reply:
Yes https://github.com/sonata-project/SonataCoreBundle/blob/3.x/src/CoreBundle/Exception/InvalidParameterException.php#L16-L18

Reply:
It doesn't say in favor of what it is deprecated, but ok!